CONTRIBUTING.md

```diff
@@ -9,6 +9,47 @@
 
 ## Setting up development environment
 
+### Preparing the development environment (Windows)
+
+Several packages & environment variables are necessary to build in a Windows
+environment:
+
+1. Download & install the latest [Go MSI][go-msi].
+
+1. Download & install [Make for Windows][make-for-windows].
+
+1. Open `cmd` and add make to the user PATH:
+
+  ```bat
+  setx path "%PATH%;C:\Program Files (x86)\GnuWin32\bin"
+  ```
+
+1. (Optional) Install [Windows Terminal][windows-terminal].
+
+1. Download & install [64-bit Git for Windows Setup][git-for-windows]. If Windows
+  Terminal was installed in the previous step, enable the
+  `Add a Git Bash Profile` component in the installer.
+
+1. Open gitbash (or Windows Terminal using gitbash shell).
+
+1. Clone this repository:
```

Comment:
I think I would assume the user has the repository already cloned, I would skip this step and straight to `cd otelcolbuilder`, the assumption being the user is already in the root of the repo. This is what is being done in the `How to build` steps below and I think it's enough.